index.bs

```diff
@@ -297,8 +318,25 @@ spec: HEADER-STRUCTURE; urlPrefix: https://httpwg.org/http-extensions/draft-ietf
       then return true.</li>
       <li>Otherwise, for each <var>item</var> in the <a>allowlist</a>:
         <ol>
-          <li>If <var>item</var> is [=same origin-domain=] with
-          <var>origin</var>, then return true.</li>
+          <li>If <var>item</var> is a tuple of an [=origin=] that's [=same origin-domain=] 
+          with <var>origin</var> and the boolean <code>False</code>, then return true.</li>
+          <li>Else if <var>item</var> is a tuple of a <a>serialized-origin</a> that's not [=same origin-domain=] 
+          with <var>origin</var> and the boolean <code>True</code>, then:</li>
```

Comment:
How can you compare a serialized-origin like that? Wouldn't it have to be parsed first?